peer-pod-controller/Makefile

```diff
@@ -150,31 +150,17 @@ undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/confi
 CONTROLLER_GEN = $(shell pwd)/bin/controller-gen
 .PHONY: controller-gen
 controller-gen: ## Download controller-gen locally if necessary.
-	$(call go-get-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen@v0.8.0)
+	GOBIN=$(shell pwd)/bin go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.10.0
```

Comment:
Hi, thanks for this patch, it's definitely a valid change. Maybe we can test if the binary already exists locally and only call go install if it's not? 

I think when you create a new controller with a recent operator-sdk it creates something like this:

controller-gen: $(CONTROLLER_GEN) ## Download controller-gen locally if necessary.
$(CONTROLLER_GEN): $(LOCALBIN)
	test -s $(LOCALBIN)/controller-gen || GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-tools/cmd/controller-gen@$(CONTROLLER_TOOLS_VERSION)

What do you think?

Reply:
@jensfr, yes. That would be better. Thanks for the suggestion. Addressed the change, PTAL.